modules/ROOT/pages/policy-mule4-tokenization.adoc

```diff
@@ -21,29 +21,37 @@ endif::[]
 
 |===
 
-[NOTE]
---
-This policy is designed to work only for Anypoint Runtime Fabric deployments.
---
+Tokenization is the process of tokenizing a value or piece of information, which might be considered sensitive data, into the token that can be mapped back to the original value. 
 
-== Tokenization
+This tokenized information replaces the original value with the token, thus minimizing the risk of matching it to the absolute value, in case a third party accesses the data. For example, this policy may be used to tokenize credit card numbers and map them to completely different ones from the originals.
 
-Tokenization is the process of tokenizing a value or piece of information (presumably sensitive data) into another one (i.e., the token), that can be mapped back onto the original value. +
-This tokenized information, replaces the original value with the token, thus minimizing the risk of matching it to the raw value in case someone happened to read it. For instance, it may be used to tokenize credit card numbers and map them to completely different ones from the originals.
+This policy is designed to work only for Anypoint Runtime Fabric deployments.
 
 == Prerequisites
 
-* To create and apply policies you need the *Manage Policies* permission.
-* You need to have a tokenization service up and running.
-** To configure and see the tokenization service, you need the *Manage Tokenization Services* permission. +
-See xref:anypoint-security::tokenization-example.adoc[Configure and Use Tokenization] for more information.
+To apply and manage the Tokenization policy, you must have:
+
+* The *Manage Policies* permission.
+* The *Manage Tokenization Services* permission to xref:anypoint-security::tokenization-example.adoc[configure] and see the tokenization service. 
+* A tokenization service up and running.
```

Comment:
@LorenaSoledad shouldn't runtime fabric be a pre-requisite here as well? At which point of the configuration workflow is an issue if you don't use Runtime Fabric?

Reply:
Mmm... Indeed, I think that RTF is a prerequisite for deploying a Tokenization Service, so maybe it's kind of transitive. But, it doesn't do harm to include it also here.

Reply:
Done! Thanks!